src/scripts/tbtcv2-rewards/rewards.ts

```diff
@@ -71,7 +71,25 @@ program
 const options = program.opts()
 const prometheusJob = options.job
 const prometheusAPI = options.api
-const clientReleases = options.releases.split("|") // sorted from latest to oldest
+
+// May 1st 2024 distribution special case:
+// Since the release of tBTC client v2.0.0, the versioning strategy has changed.
+// Previous versions had a -mX suffix, but this is no longer the case.
+// From now on, the versioning system is semantic versioning https://semver.org/
+// without suffixes, where it is mandatory the update of the client for new minor
+// versions, but optional for new patch versions.
```

Comment:
Non-blocking:

English is not my first language, but this sentence sounds a little bit strange to me and might not be clear enough 🤔 What do you think about this:

```suggestion
// without suffixes, where it is mandatory to update the client for new minor
// versions, but optional for new patch versions.
```

or 

```
(...) where the update of the client is mandatory for new minor versions, but optional for new patch versions.
```

Reply:
Done!